core/src/main/scala/cromwell/core/simpleton/WomValueBuilder.scala

```diff
@@ -224,10 +224,10 @@ object WomValueBuilder {
   private case class SimpletonComponent(path: String, value: WomValue)
 
   def toJobOutputs(taskOutputs: Traversable[OutputPort], simpletons: Traversable[WomValueSimpleton]): CallOutputs = {
-    CallOutputs(toWdlValues(taskOutputs, simpletons))
+    CallOutputs(toWomValues(taskOutputs, simpletons))
   }
 
-  def toWdlValues(taskOutputs: Traversable[OutputPort], simpletons: Traversable[WomValueSimpleton]): Map[OutputPort, WomValue] = {
+  def toWomValues(taskOutputs: Traversable[OutputPort], simpletons: Traversable[WomValueSimpleton]): Map[OutputPort, WomValue] = {
```

Comment:
As long as you're in here doing this there are many more opportunities to remove inappropriate references to `WDL` in this file. 😄